### PR TITLE
Add features: Rules, Restart, Volume, and Red Outlines.

### PR DESCRIPTION
**Pull request by xproot**
This pull requests consists of four new features conveniently separated into four commits. These are quite helpful that I would go as far to say this improves QoL.

- Rules button, which will show a iFrame to Help.htm, which will explain the rules of JSBingoSlots to the player.
- Restart button, which will allow the player to end the game mid-game.
- Volume button, which will allow the player to mute in-game audio.
- "Flashes" of a red outline, when points or free spins are modified, these will show.

As it stands right now, this is what the uses sees when going into a new game:
image

Right here you can see three of the four updates I've made:
image

## Rules
Clicking on the Rules button will show an iFrame pointing to Help.htm
image

## Restart
Mid-game, this will allow you to instantly end the game.
image

## Volume
Instantly mutes all DOM audio.
image

## Red Outlines
When scores and free spins are updated, they will show a red outline.
image
image

### CODE REVIEW

File: slingogame.js
Comment (xproot) [inline slingogame.js:1]: The diffs GitHub gives fries my brain, I just moved some code around due to the restart btn lol.

**Comment by xproot** [comment]
Nick JSBingoSlots

**Comment by xproot** [comment]
https://github.com/nick99nack/JSBingoSlots/assets/49620652/61e67599-3b26-4fa8-9644-d868813797f7

**Comment by xproot** [comment]
https://github.com/nick99nack/JSBingoSlots/assets/49620652/347f9f78-7102-4abd-a4b5-84ab3bf2e8d8

**Comment by seal331** [comment]
there is a bug somewhere in this code and xp won't fix it according to discord messages

**Comment by xproot** [comment]
> there is a bug somewhere in this code and xp won't fix it according to discord messages

tf you mean

**Comment by xproot** [comment]
> there is a bug somewhere in this code and xp won't fix it according to discord messages

tf you mean (x2)

**Pull request merged**